gpt4all-bindings/python/gpt4all/gpt4all.py

```diff
@@ -312,8 +310,8 @@ def get_model_from_name(model_name: str) -> pyllmodel.LLModel:
             return pyllmodel.LlamaModel()
         elif model_name in MPT_MODELS:
             return pyllmodel.MPTModel()
-        else:
-            err_msg = f"""No corresponding model for provided filename {model_name}.
-            If this is a custom model, make sure to specify a valid model_type.
-            """
-            raise ValueError(err_msg)
+
+        err_msg = f"""No corresponding model for provided filename {model_name}.
+        If this is a custom model, make sure to specify a valid model_type.
+        """
```

Comment:
maybe instead, do:
```py
       err_msg = (f"No corresponding model for provided filename {model_name}.\n"
                   "If this is a custom model, make sure to specify a valid model_type.\n")
```
Otherwise you get the full indent on the second line. There might be some functionality somewhere that deals with that for docstrings, I think? But I forgot where that was.

Reply:
Makes sense.

> There might be some functionality somewhere in the standard library that deals with that for docstrings, I think? But I forgot where that was.

There's `textwrap.dedent()`, but I rarely use it because it's not widely understood... Combining strings looks good enough